external/PATCH-01-STRATUM

```diff
@@ -3829,32 +3845,29 @@ index 9f4bb3a9..e0154d94 100644
 +}
 +
 +::util::Status TdiSdeWrapper::AddPort(int device, int port, uint64 speed_bps,
-                                      FecMode fec_mode) {
++                                     FecMode fec_mode) {
 +  auto port_attrs = absl::make_unique<port_attributes_t>();
 +#ifdef P4SDE_DPDK_TARGET_STUB
-   ASSIGN_OR_RETURN(auto bf_speed, PortSpeedHalToBf(speed_bps));
-   ASSIGN_OR_RETURN(auto bf_fec_mode, FecModeHalToBf(fec_mode, speed_bps));
--  RETURN_IF_BFRT_ERROR(bf_pal_port_add(static_cast<bf_dev_id_t>(device),
```

Comment:
Why is this being removed?

Reply:
Please check if the new patch file has this line present -- maybe the lines have just shifted in patch file and diff cannot show properly.